src/body.rs

```diff
@@ -175,6 +175,9 @@ pub trait LineMetaMut: LineMeta {
     fn set_labels(&mut self, labels: KeyValueMap) -> Result<(), LineMetaError>;
     fn set_level(&mut self, level: String) -> Result<(), LineMetaError>;
     fn set_meta(&mut self, meta: Value) -> Result<(), LineMetaError>;
+    fn get_line(&mut self) -> (Option<&str>, Option<&[u8]>);
```

Comment:
get_line could be edited to just return &[u8] which is valid for Vec<u8>, String and Bytes, etc. I think it makes more sense to treat log lines as bytes in general as we don't make any assumptions about their content except that we can discard non-utf8 bytes when serializing, and we might change that in future to be able to ship non-utf8 strings